tests/backup/backup_helper.go

```diff
@@ -1218,85 +1089,104 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 }
 
 // backupSuccessCheck inspects backup task
-func backupSuccessCheck(backupName string, orgID string, retryDuration int, retryInterval int, ctx context.Context) (bool, error) {
-	backupDriver := Inst().Backup
+func backupSuccessCheck(backupName string, orgID string, retryDuration time.Duration, retryInterval time.Duration, ctx context.Context) error {
 	if retryDuration == 0 {
-		retryDuration = maxWaitPeriodForBackupCompletionInMinutes
+		retryDuration = maxWaitPeriodForBackupCompletionInMinutes * time.Minute
 	}
 	if retryInterval == 0 {
-		retryInterval = 30
+		retryInterval = 30 * time.Second
```

Comment:
Let us not do this. Who ever is calling the backup success check should pass the exact timeout period and retry time explicitly. Providing `0` and then setting it to a default time does not give the correct picture.

Reply:
Also, I would suggest `maxWaitPeriodForBackupCompletionInMinutes` should be made 30 minutes

Reply:
Done.